### DISCUSSION TITLE
这是不是因为chromium下载失败了

### DISCUSSION THREAD

**Discussion by Kosue02** [Arondight/Adachi-BOT#17] [Q&A] [answered]
Error: Failed to launch the browser process!
/root/Adachi-BOT/node_modules/puppeteer/.local-chromium/linux-848005/chrome-linux/chrome: error while loading shared libraries: libatk-1.0.so.0: cannot open shared object file: No such file or directory

**Comment by Arondight** [comment, marked as answer]
这应该是系统内缺少库文件，没有下载失败。

你可以用系统内Chromium 试试，至少系统内的包会自己解决依赖关系，详情看README。
